This one (plus a misrule in my pf.conf) made my box crash at random after 6
to 48 hours of uptime. Thanks to an incoming icmp packet.

Taken from:
http://svn.freebsd.org/viewvc/base/head/sys/netinet/if_ether.c?r1=198418&r2=201416
--- head/sys/netinet/if_ether.c
+++ head/sys/netinet/if_ether.c
@@ -372,8 +372,8 @@
 	if (la->la_asked < V_arp_maxtries)
 		error = EWOULDBLOCK;	/* First request. */
 	else
-		error =
-			(rt0->rt_flags & RTF_GATEWAY) ? EHOSTUNREACH : EHOSTDOWN;
+		error = rt0 != NULL && (rt0->rt_flags & RTF_GATEWAY) ?
+		    EHOSTUNREACH : EHOSTDOWN;
 
 	if (renew) {
 		LLE_ADDREF(la);
